src/widget.ts

```diff
@@ -104,36 +131,37 @@ export class DeephavenView extends DOMWidgetView {
     }
   };
 
-  //@ts-ignore
-  onDisconnect = (type): void => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  onRestartOrTerminate = (sender: any, args: string) => {
```

Comment:
You could use `unknown` here instead and get rid of the warning, since you're not using `sender`:
```suggestion
  onRestartOrTerminate = (sender: unknown, args: string) => {
```